recipes/jupyterlab-lsp/meta.yaml

```
@@ -0,0 +1,115 @@
+{% set name = "jupyter-lsp" %}
+{% set ext_name = "@krassowski/jupyterlab-lsp" %}
+{% set version = "0.8.0" %}
+{% set build_number = "0" %}
+
+{% set py_spec = ">=3.5" %}
+{% set lab_spec = ">=2,<3.0.0a0" %}
+{% set ext_spec = "@^1.0.0" %}
+
+# List of jupyter_lsp py.tests ran for each known servers
+{% set server_tests = ["serverextension", "start_known", "listeners"] %}
+
+package:
+  name: {{ name }}-suite  # To avoid issue https://github.com/conda/conda-build/issues/3933
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3abe20ec48a3f6b132653abe597af2b6cd0005539ab0d74e0ba203927124d333
+
+build:
+  noarch: python
+  number: {{ build_number }}
+
+outputs:
+  - name: {{ name }}
+    build:
+      number: {{ build_number }}
+      noarch: python
+    script: build_base.bat  # [win]
+    script: build_base.sh  # [not win]
```

No need for the script files.
```suggestion
    script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```